tensorflow/lite/kernels/conv_test.cc

```diff
@@ -227,6 +225,210 @@ TEST_P(ConvolutionOpTest, InputAndFilterSameWidthHeight) {
+  });
+  m.SetBias({1, 2, 3});
+
+  m.Invoke();
+
+  EXPECT_THAT(m.GetOutput(), ElementsAreArray({
+                                 18, 2, 5,  // first batch, left
+                                 18, 2, 5,  // first batch, right
+                                 17, 4, 3,  // second batch, left
+                                 37, 4, 3,  // second batch, right
+                             }));
+}
+
+TEST_P(ConvolutionOpTest, ActivationSignTest) {
+  ConvolutionOpModel m(
+      GetRegistration(), {TensorType_FLOAT32, {2, 2, 4, 1}},
+      {TensorType_FLOAT32, {3, 2, 2, 1}}, {TensorType_FLOAT32, {}},
+      /*stride_width*/ 2,
+      /*stride_height*/ 2,
+      /*Padding*/ Padding_VALID,
+      /*ActivationFunctionType*/ ActivationFunctionType_SIGN_BIT);
+
+  m.SetInput({
+      // First batch
+      1, 1, 1, 1,  // row = 1
+      2, 2, 3, 2,  // row = 2
+      // Second batch
+      1, 2, 3, 4,  // row = 1
+      1, 2, 4, 4,  // row = 2
+  });
+  m.SetFilter({
+      1, 2, 3, 4,    // first 2x2 filter
+      -1, 1, -1, 1,  // second 2x2 filter
+      -1, -1, 1, 1,  // third 2x2 filter
+  });
+  m.SetBias({1, 2, 3});
+
+  m.Invoke();
+
+  EXPECT_THAT(m.GetOutput(), ElementsAreArray({
+                                 18, 2, 5,  // first batch, left
+                                 21, 1, 6,  // first batch, right
+                                 17, 4, 3,  // second batch, left
+                                 40, 3, 4,  // second batch, right
+                             }));
+}
+
+TEST_P(ConvolutionOpTest, StrideTest) {
+  ConvolutionOpModel m(
+      GetRegistration(), {TensorType_FLOAT32, {2, 2, 4, 1}},
+      {TensorType_FLOAT32, {3, 2, 2, 1}}, {TensorType_FLOAT32, {}},
+      /*stride_width*/ 1,
+      /*stride_height*/ 1,
+      /*Padding*/ Padding_VALID,
+      /*ActivationFunctionType*/ ActivationFunctionType_SIGN_BIT);
+
+  m.SetInput({
+      // First batch
+      1, 1, 1, 1,  // row = 1
+      2, 2, 3, 2,  // row = 2
+      // Second batch
+      1, 2, 3, 4,  // row = 1
+      1, 2, 4, 4,  // row = 2
+  });
+  m.SetFilter({
+      1, 2, 3, 4,    // first 2x2 filter
+      -1, 1, -1, 1,  // second 2x2 filter
+      -1, -1, 1, 1,  // third 2x2 filter
+  });
+  m.SetBias({1, 2, 3});
+
+  m.Invoke();
+
+  EXPECT_THAT(m.GetOutput(), ElementsAreArray({
+                                 18, 2, 5,  // first batch, left
+                                 22, 3, 6,  // first batch, right
+                                 21, 1, 6,  // second batch, left
+                                 17, 4, 3,  // second batch, right
+                                 31, 5, 4,  // second batch, right
+                                 40, 3, 4,  // second batch, right
+                             }));
+}
+
+TEST_P(ConvolutionOpTest, PaddingTest) {
+  ConvolutionOpModel m(
+      GetRegistration(), {TensorType_FLOAT32, {1, 2, 4, 1}},
+      {TensorType_FLOAT32, {3, 2, 2, 1}}, {TensorType_FLOAT32, {}},
+      /*stride_width*/ 1,
```

Comment:
Nit: Prefer /*stride_width=*/ as the descriptor for consistency, same with all other commented args.

Reply:
@jdduke , thanks for the comments, i have updated the code as per your suggestion, kindly check and approve.

Regards
Amit